src/Security/Authorization/Policy/src/AuthorizationMiddleware.cs

```diff
@@ -17,11 +18,18 @@ public class AuthorizationMiddleware
 
         private readonly RequestDelegate _next;
         private readonly IAuthorizationPolicyProvider _policyProvider;
+        private readonly AuthorizationMiddlewareOptions _options;
 
-        public AuthorizationMiddleware(RequestDelegate next, IAuthorizationPolicyProvider policyProvider)
+        public AuthorizationMiddleware(RequestDelegate next, IAuthorizationPolicyProvider policyProvider) 
+            : this(next, policyProvider, new OptionsWrapper<AuthorizationMiddlewareOptions>(new AuthorizationMiddlewareOptions())) 
+        { 
+        }
+        
+        public AuthorizationMiddleware(RequestDelegate next, IAuthorizationPolicyProvider policyProvider, IOptions<AuthorizationMiddlewareOptions> options)
```

Comment:
Did we ever fix the ActivatorUtils ordering bug where only the first constructor in the file would be called?

Reply:
You can use https://docs.microsoft.com/en-us/dotnet/api/microsoft.extensions.dependencyinjection.activatorutilitiesconstructorattribute.-ctor?view=dotnet-plat-ext-3.1 to influence the constructor being used.